packages/util/src/permissions/constants.ts

```
@@ -0,0 +1,42 @@
+{
+  {
+    /**
+     * Constants and types for KlickerUZH permissions:
+     * - UserAccessMap: maps user IDs to { maxAccessLevel, parentPermissionId, derived }.
+     * - permissionLevelMap: defines numeric ordering for PermissionLevel enums.
+     * - inversePermissionLevelMap: maps numeric levels back to PermissionLevel.
+     */
+  }
+}
```

Biome (1.9.4)

[error] This block statement doesn't serve any purpose and can be safely removed.

Standalone block statements without any block-level declarations are redundant in JavaScript and can be removed to simplify the code.

(lint/complexity/noUselessLoneBlockStatements)